src/sentry/api/serializers/models/organization.py

```diff
@@ -143,18 +143,23 @@ class _Status(TypedDict):
     name: str
 
 
+class _Links(TypedDict):
+    organizationUrl: str
+    regionUrl: str
```

Comment:
Should this have sentryUrl too?

Reply:
@markstory I didn't add it because the frontend POC hasn't made use of it. I planned to add `sentryUrl` here once I've found a need for it.